### ISSUE TITLE
Fails to build for i386 with gcc-13

### ISSUE THREAD

**Issue by bwhacks** [oracle/ktls-utils#18]
A CI build for i386 failed with the following error:

```
config.c: In function 'tlshd_config_read_datum':
config.c:155:52: error: comparison of integer expressions of different signedness: '__off_t' {aka 'long int'} and 'unsigned int' [-Werror=sign-compare]
  155 |         if (statbuf.st_size < 0 || statbuf.st_size > UINT_MAX) {
      |                                                    ^
```

The full build log is at <https://salsa.debian.org/kernel-team/ktls-utils/-/jobs/4457752>.

**Comment by chucklever** [comment]
This one should already be addressed by commit 52ac9ff05a5e ("tlshd: fix max config file size comparison") in 0.10-dev. Feel free to re-open this issue if that is not the case.

In the meantime, it would be reasonable to add a GitHub action for build-testing on i386 and maybe one or two other commonly-used platforms. I will see what can be done.

**Comment by bwhacks** [comment]
> This one should already be addressed by commit 52ac9ff ("tlshd: fix max config file size comparison") in 0.10-dev. Feel free to re-open this issue if that is not the case.

Yes, sorry I didn't check that.
